CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
-OrionHDFSSink parameters are now called as "hdfs_*" instead of "cosmos_*" (#374)
+[FEATURE] OrionHDFSSink parameters are now called as "hdfs_*" instead of "cosmos_*" (#374)
```

Comment:
Not important but it sounds better to me: '... are now called "hdfs_*"...' :) But I leave it to your discretion ;)